.github/workflows/release-backend.yml

```diff
@@ -0,0 +1,13 @@
+on:
+  push:
+    tags:
+      - ccdscan-fungus/*
+
+jobs:
+  release-ccdscan-fungus:
+    uses: concordium/.github/.github/workflows/docker-release-workflow.yaml@v1
```

Comment:
It has been mentioned that we would rather do copy paste. I would for now rather have it like this so we do not diverge in the repository

Reply:
It adds another level of complexity because people have to check out that above file and figure out what it does (since it is written generically while we only need a few lines of code here to push to docker).